tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Services/CreateOrderItemServiceTests.cs

```diff
@@ -21,5 +35,243 @@ public static void Constructors_VerifyGuardClauses()
 
             assertion.Verify(constructors);
         }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static void Constructor_NullDbContext_ThrowsArgumentNullException(
+            OrderItemValidator orderItemValidator,
+            ServiceRecipientService serviceRecipientService)
+        {
+            Assert.Throws<ArgumentNullException>(() => _ = new CreateOrderItemService(null, orderItemValidator, serviceRecipientService));
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static void Constructor_NullOrderItemValidator_ThrowsArgumentNullException(
+            ApplicationDbContext context,
+            ServiceRecipientService serviceRecipientService)
+        {
+            Assert.Throws<ArgumentNullException>(() => _ = new CreateOrderItemService(context, null, serviceRecipientService));
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static void Constructor_NullServiceRecipient_ThrowsArgumentNullException(
+            ApplicationDbContext context,
+            OrderItemValidator orderItemValidator)
+        {
+            Assert.Throws<ArgumentNullException>(() => _ = new CreateOrderItemService(context, orderItemValidator, null));
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static async Task CreateAsync_InvalidAggregateValidation_ValidateResult(
```

Comment:
I think something like `CreateAsync_InvalidAggregateValidation_SuccessIsFalse` might be a slightly better name.

Reply:
Renamed test method to `CreateAsync_InvalidAggregateValidation_SuccessIsFalse`